app/assets/javascripts/curation_concerns/file_manager/sorting.es6

```diff
@@ -82,4 +87,40 @@ export default class SortManager {
       }
     ).toArray()
   }
+
+  get alpha_sort_button() {
+    return $("*[data-action='alpha-sort-action']")
+  }
+
+  initialize_alpha_sort_button() {
+    Blacklight.onLoad(() => {
+      let that = this
+      this.alpha_sort_button.click(function() { that.sort_alpha() } )
+    })
+  }
+
+  sort_alpha() {
+    // create array of { title, element } objects
+    let array = []
+    let children = this.element.children().get()
+    children.forEach(function(child) {
+      let title = $(child).find("input.title").val()
+      array.push(
+        { title: title,
+          element: child }
+      )
+    })
+    // sort array by title of each object
+    array.sort(function(o1, o2) {
+      let a = o1.title.toLowerCase()
+      let b = o2.title.toLowerCase()
+      return a < b ? -1 : (a > b ? 1 : 0);
+    });
+    // replace contents of #sortable with elements from the array
+    this.element.empty()
```

Comment:
Does sorting still work after you re-order this way? I'm not sure how grumpy jquery-ui-sortable gets if you delete its container contents.